src/lxc/start.c

```diff
@@ -850,6 +849,12 @@ static int do_start(void *data)
 		INFO("Unshared CLONE_NEWCGROUP.");
 	}
 
+	/* Setup the container, ip, names, utsname, ... */
+	if (lxc_setup(handler)) {
```

Comment:
Ok, so I confirmed that `pivot_root()` and `CLONE_NEWCGROUP` do not hate each other. :) That's a good sign.